aiida_quantumespresso/parsers/parse_xml/pw/parse.py

```diff
@@ -340,6 +305,18 @@ def parse_pw_xml_post_6_2(xml, include_deprecated_v2_keys=False):
     # Band structure
     if 'band_structure' in outputs:
         band_structure = outputs['band_structure']
+
+        smearing_xml = None
+
+        if 'smearing' in outputs['band_structure']:
+            smearing_xml = outputs['band_structure']['smearing']
+        elif 'smearing' in inputs:
+            smearing_xml = inputs['smearing']
+
+        if smearing_xml:
+            xml_data['degauss'] = smearing_xml['@degauss']
+            xml_data['smearing_type'] = smearing_xml['$']
```

Comment:
I guess this is one of the first cases where we are really changing the behaviour w.r.t. the old parser, right?

- Can we add already now a list of things that changed between version 2 and 3 (or even more precisely, that depend on how QE was compiled)? Possibly the list is already there, we should then just add also this

- I don't see any unit conversion - normally, in the parsed output, we try to use consistently `eV` for energy

Reply:
I have started a list in issue #384 . The units are not converted because it is not clear what they are. I sent an email for this to Delugas and put you in CC. The docs and file itself say it should be Hartree, but it is clearly Rydberg.

Reply:
Ok, let's add this to the issue, and let's wait for the units (or add conversion from Rydberg since that's what they are and if they fix this, this will be only for future versions)

Reply:
I performed a calculation for 6.4, 6.4.1 and 6.5 and only 6.5 has the correct units. So for now I have just added the correct unit conversion in the parser. Everything 6.5 and above is assumed Hartree, for all version below I assume it is Rydberg. So @giovannipizzi this should be ready for review